webpack.config.ts

```diff
@@ -86,7 +86,15 @@ const config: CallableOption = (_env, argv) => ({
           from: path.join(__dirname, "node_modules/@fontsource-variable/roboto-flex/index.css"),
           to: path.join(OUT_PATH, "font.css"),
           transform: (input: Buffer) => {
-            return input.toString().replace(/url\(.\/files\//g, "url(./fonts/");
+            return input.toString().replace(/url\(.\/files\//g, "url(./fonts/").concat("\n\n/* vazirmatn-arabic-wght-normal */\n" +
+                "@font-face {\n" +
+                "  font-family: 'Vazirmatn Variable';\n" +
+                "  font-style: normal;\n" +
+                "  font-display: swap;\n" +
+                "  font-weight: 100 900;\n" +
+                "  src: url(./fonts/vazirmatn-arabic-wght-normal.woff2) format('woff2-variations');\n" +
+                "  unicode-range: U+0600-06FF,U+0750-077F,U+0870-088E,U+0890-0891,U+0898-08E1,U+08E3-08FF,U+200C-200E,U+2010-2011,U+204F,U+2E41,U+FB50-FDFF,U+FE70-FE74,U+FE76-FEFC,U+102E0-102FB,U+10E60-10E7E,U+10EFD-10EFF,U+1EE00-1EE03,U+1EE05-1EE1F,U+1EE21-1EE22,U+1EE24,U+1EE27,U+1EE29-1EE32,U+1EE34-1EE37,U+1EE39,U+1EE3B,U+1EE42,U+1EE47,U+1EE49,U+1EE4B,U+1EE4D-1EE4F,U+1EE51-1EE52,U+1EE54,U+1EE57,U+1EE59,U+1EE5B,U+1EE5D,U+1EE5F,U+1EE61-1EE62,U+1EE64,U+1EE67-1EE6A,U+1EE6C-1EE72,U+1EE74-1EE77,U+1EE79-1EE7C,U+1EE7E,U+1EE80-1EE89,U+1EE8B-1EE9B,U+1EEA1-1EEA3,U+1EEA5-1EEA9,U+1EEAB-1EEBB,U+1EEF0-1EEF1;\n" +
+                "}");;
```

Comment:
As Arne already commented, this is not ideal here. At the very least, the big CSS part should be put into a separate CSS file and not specified as big string literal here. But I also don't quite like asymmetry of starting with one of the two input CSS files and adding the other one to it. 

This is a big finicky but I think I have an idea that's decent:

First, it's worth noting that the unicode ranges for the latin and latin-ext versions of "vazirmatn" are the same or a subset of the ranges from Roboto. That means that we can just use the full `node_modules/@fontsource-variable/vazirmatn/index.css` file. The entries regarding the latin and latin-ext versions are ignored since a higher priority font already defined all these unicode codepoints. So the browser will, as far as I can tell, never attempt to load the vazirmatn latin(-ext) versions. 

Second, I found out about the `transformAll` option for the webpack copy plugin which allows you to combine multiple files into one. So with that, I think sth along these lines should work (I hope I understood the glob syntax correctly):

```ts
{
  from: path.join(__dirname, "node_modules/@fontsource-variable/@(roboto-flex|vazirmatn)/index.css"),
  to: path.join(OUT_PATH, "font.css"),
  transformAll: ...
}
```